.changeset/config.json

```diff
@@ -7,5 +7,9 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": [],
+  "privatePackages": {
+    "version": true,
+    "tag": true
+  }
```

Comment:
Another attempt at getting this to return true

https://github.com/wp-graphql/wpgraphql-ide/blob/main/.github/workflows/release-plugin.yml#L39